contracts/ExitFormat.sol

```diff
@@ -48,6 +48,14 @@ library ExitFormat {
         return abi.encode(exit);
     }
 
+    function decodeGuaranteeData(bytes memory data)
+        internal
+        pure
+        returns (address[] memory)
+    {
+        return abi.decode(data, (address[]));
+    }
```

Comment:
This is good but perhaps belongs in `Nitro.sol`? We want to keep this file as protocol agnostic as possible.

Reply:
7298f49